### PR TITLE
Newsletters: add 187 (2022-02-16)

### PR DESCRIPTION
**Pull request by harding**
- Client/services section @bitschmidty 
- Eclair 2164 @dongcarl 
- Other stuff @harding 

(Sorry, there are probably problems with this PR.  My laptop broke and I haven't yet restored my ability to preview the newsletter locally.)

### CODE REVIEW

**Comment by Xekyo** [comment]
@harding: Anything we can do to help?

File: _posts/en/newsletters/2022-02-16-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+- **Simplified alternative to `OP_TXHASH`:** in continuing discussion
+  about covenant-enabling opcodes (see [Newsletter #185][news185
+  composable]), Rusty Russell [proposed][russell op_tx] that the
+  function provided by `OP_TXHASH` could be provided by the existing
+  `OP_SHA256` opcode plus a new `OP_TX` opcode which accepted the same
+  input as `OP_TXHASH`.  The new opcode would make serialized fields
+  from a spending transaction available to a [tapscript][topic
+  tapscript].  Scripts could then test the fields directly (e.g. is the
+  transaction version's number greater than `1`?) or hash the data and
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-02-16-newsletter.md:22]:
```suggestion
  transaction's version number greater than `1`?) or hash the data and
  ```

**Review by Xekyo — COMMENTED** [review]
I edited the title of the PR to amend the date, but the commit message has still the wrong issue and date.

**Comment by Xekyo** [comment]
I built the site locally, everything looks alright, and all links included so far resolve to the correct site.

**Comment by Xekyo** [comment]
I updated the commit message to reference the correct issue and date.

**Review by bitschmidty — COMMENTED** [review]
Just one suggestion so far

**Comment by harding** [comment]
Made suggested edit (thanks @bitschmidty!) and reviewed edits and additions by @Xekyo, @bitschmidty, and @dongcarl (thanks!), all of which look great to me.  Removed releases/RC section (nothing this week) and added topic links.

Thanks everyone, doubly so for the extra help while I was have tech issues!

File: _posts/en/newsletters/2022-02-16-newsletter.md [hunk 2]
```diff
@@ -57,10 +57,10 @@ wallets and services.*
   performance] in the [1.6.0 release][fulcrum 1.6.0].
 
 - **Kraken announces proof of reserves scheme:**
-  [Kraken details][kraken por] their proof of reserves scheme involving a
+  Kraken [details][kraken por] their proof of reserves scheme involving a
   trusted auditor, also noting shortcomings and future improvements. Kraken
-  proves onchain address ownership by digitally signing,
-  a merkle tree of Kraken user account balances is created, the auditor verifies
+  proves onchain address ownership by digitally signing
```
Comment (bitschmidty) [inline _posts/en/newsletters/2022-02-16-newsletter.md:62]: @harding Im not sure this is right. I believe Kraken is signing to prove ownership of UTXOs onchain. Not signing a merkle tree

Comment (harding) [reply _posts/en/newsletters/2022-02-16-newsletter.md]: Sorry!  Hopefully fixed in the latest push.

Comment (Xekyo) [reply _posts/en/newsletters/2022-02-16-newsletter.md]: That mention of Kraken's POR made me wonder how they show each user separate UTXOs, and especially how they prove that. It doesn't seem particularly scalable to maintain each user's balance as distinct UTXOs.

Comment (Xekyo) [reply _posts/en/newsletters/2022-02-16-newsletter.md]: Ah nevermind. Rereading that section, I realize now that the user balances on the exchange was included in the total and the notary certifies that the on-chain balance exceeds all deposits. It's much clearer now!

**Review by Xekyo — COMMENTED** [review]
Looks good to me in another pass! I can rebuild it locally when I get home later and check the links again.

File: _posts/en/newsletters/2022-02-16-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Eclair #2164][] improves its handling of feature bits in various contexts.
+  Notably, invoices that requires a mandatory but non-invoice feature would no
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-02-16-newsletter.md:77]:
```suggestion
  Notably, invoices that require a mandatory but non-invoice feature would no
```

**Comment by glozow** [comment]
LGTM other than the s/requires/require - thanks!

**Comment by bitschmidty** [comment]
Pushed a commit for @adamjonas feedback, ensured local build/checks were successful, squashed and merged.

Thanks @harding @dongcarl @Xekyo @glozow @adamjonas !

**Pull request merged**